### ISSUE TITLE
Add platform: Invalid or unexpected token

### ISSUE THREAD

**Issue by codi0** [manifoldjs/ManifoldCordova#115]
I get the error "Invalid or unexpected token" in the CLI after installing the hostedwebapp plugin (disappears on removal), when adding any of android, ios or browser platforms to cordova.

I've tried removing the engine constraints in plugin.xml and using different platform versions (e.g. android 8.0.0 and 5.1.1), but the error is always the same and prevents any build / run commands from executing.

Unfortunately I'm not sure how to debug the plugin error on cordova. Any advice?

**Comment by fikkatra** [comment]
Ran into this issue too, running the command with '-d' (for verbose logging) indicated that the issue was the shebang
`#!/usr/bin/env node` 
in updateConfigurationBeforePrepare.js. Downgrading to node 8 fixed it, but there should be a better way :(.

**Comment by JudahGabriel** [comment]
Yes, currently the CLI and its pwabuilder-lib are stuck on an older version of Node due to 3rd party dependencies monkey-patching the Node file system APIs. We're working to fix it so that we can run on newer Node version.
